### PR TITLE
Add tn options

### PR DESCRIPTION
**Pull request by yzhang123**

### CODE REVIEW

File: nemo_text_processing/text_normalization/en/data/address/states.tsv [hunk 1]
```diff
@@ -28,8 +29,12 @@ Nebraska	NE
 Nevada	NV
 New Hampshire	NH
 New Jersey	NJ
+New Jersey	N.J
+New Jersey	N. J
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/data/address/states.tsv:33]: maybe make a general rule so various formats are. supported for other states as well?

File: nemo_text_processing/text_normalization/en/data/measurements.tsv [hunk 2]
```diff
@@ -53,6 +53,7 @@ atm	atmosphere
 db	decibel
 ps	peta second
 oz	ounce
+fl	florida
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/data/measurements.tsv:56]: could you remind the sentence you encountered this? [I can't find such measure unit ](https://www.merriam-webster.com/dictionary/fl)

File: nemo_text_processing/text_normalization/en/taggers/cardinal.py [hunk 3]
```diff
@@ -155,8 +155,8 @@ def get_serial_graph(self):
         """
         num_graph = self.single_digits_graph
 
-        if not self.deterministic:
-            num_graph |= self.graph
+        # if not self.deterministic:
+        num_graph |= self.graph
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/taggers/cardinal.py:159]: do tests pass in SH?

File: nemo_text_processing/text_normalization/en/taggers/roman.py [hunk 4]
```diff
@@ -44,16 +41,20 @@ def __init__(self, deterministic: bool = True):
...
+        names |= pynini.string_map(female_labels).optimize()
 
         # roman numerals from I to IV with a preceding name are converted to ordinal form
         graph = (
             pynutil.insert("key_the_ordinal: \"")
             + names
             + pynutil.insert("\"")
             + pynini.accep(" ")
-            + pynini.compose(pynini.union("I", "II", "III", "IV", "V", "VI", "VII"), default_graph)
+            + pynini.compose(pynini.closure(NEMO_ALPHA, 1, 5), default_graph)
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/taggers/roman.py:57]: basically no restriction for the number - could you update the comment above the `graph`?

Comment (yzhang123) [reply nemo_text_processing/text_normalization/en/taggers/roman.py:57]: ok

File: nemo_text_processing/text_normalization/en/data/measurements.tsv (hunk 2 above)
Comment (yzhang123) [reply nemo_text_processing/text_normalization/en/data/measurements.tsv:56]:
```
------------------------------
INPUT : The terms differed little from those of Breda, except that the Republic undertook to pay a war indemnity of 2,000,000 fl. within three years.
GT    : The terms differed little from those of Breda, except that the Republic undertook to pay a war indemnity of two million florida. within three years.
WFST  : ["The terms differed little from those of Breda, except that the Republic undertook to pay a war indemnity of two million fl. within three years."]

Comment (yzhang123) [reply nemo_text_processing/text_normalization/en/data/measurements.tsv:56]: ok will remove, since this is weird

**Pull request merged**